1_The_Physical/Guess_Point_Level_1_Stateless/2_Arrange_Act_Assert/2_1_Password_Validator/src/index.spec.ts

```diff
@@ -38,5 +38,16 @@ describe('password validator', () => {
         expect(validated.result).toEqual(false)
     })
 
+    it.each([
+        'Password123123123',
+        'anotherTooL0ngPswd',
+        'yepItsStillT000L00ng',
+    ])('sets result to false and error to "too short" when password is more than 15 characters (%s)', (password) => {
+        const validator = new PasswordValidator()
```

Comment:
Howdy @acsauk, just found a snag in here.

Looks like you still have "too short" as a part of the test name for this one which is validating against "too long".

You can avoid this by Encapsulating What Varies.

Since the Validation Error varies, I'd include that as a part of your test input like so:

```typescript
it.each([
        ['Password123123123', 'too short'],
        ['anotherTooL0ngPswd', 'too short'],
        ['yepItsStillT000L00ng', 'too short']
    ])('knows that %s is invalid because it is %s', (password, validationReason)
```

I believe the Jest it.each API is something like that.